apps/ex_nvr_web/lib/ex_nvr_web/user_auth.ex

```diff
@@ -100,6 +110,17 @@ defmodule ExNVRWeb.UserAuth do
     assign(conn, :current_user, user)
   end
 
+  defp fetch_username_from_header(conn), do: conn |> get_req_header("username") |> List.first()
+  defp fetch_password_from_header(conn), do: conn |> get_req_header("password") |> List.first()
```

Comment:
Where did you get this style of authentication ??
I thought the camera will use Basic authentication.

Reply:
In the issue it was mentioned `The endpoint should accept username/password basic authentication as a starter` doesn't that mean username/password mentioned in the header ?

Reply:
I said `Basic authentication`, check the basic auth header format to see that it's not the same the way you did it.
The username/password are sent as base64 encoded string in the `Authorization header`
```
Authorization: Basic <base64 encoded string>
```
When you base64 decode the string it'll be in the format `<username>:<password>`. The username/password are separated by a colon